metrics/collectors/average.lua

```diff
@@ -4,7 +4,7 @@ local Shared = require('metrics.collectors.shared')
 
 --- Collector to produce count and average value metrics.
 -- Average value is is calculated between two subsequent `:collect` calls.
-local Average = Shared:new_class('average')
+local Average = Shared:new_class('average', {'observe_latency'})
```

Comment:
I don't get this change. Is it really needed?

Reply:
It adds observe_latency to specific collector classes (histogram and average).

Reply:
Yes, thanks for explanation